callautomation-outboundcalling/main.py

```diff
@@ -73,33 +102,47 @@ def callback_events_handler():
             recording_id = recording_properties.recording_id
 
             app.logger.info("Starting recognize")
-            target_participant = PhoneNumberIdentifier(TARGET_PHONE_NUMBER)
-            main_menu_prompt = FileSource(MAIN_MENU_PROMPT_URI)
-            call_connection_client.start_recognizing_media(input_type=RecognizeInputType.DTMF,
-                                                           target_participant=target_participant,
-                                                           play_prompt=main_menu_prompt,
-                                                           interrupt_prompt=True,
-                                                           initial_silence_timeout=10,
-                                                           dtmf_max_tones_to_collect=1)
+           
+            get_media_recognize_choice_options(
+                call_connection_client=call_connection_client,
+                text_to_play=MainMenu, 
+                target_participant=target_participant,
+                choices=get_choices(),context="")
+            
 
         # Perform different actions based on DTMF tone received from RecognizeCompleted event
         elif event.type == "Microsoft.Communication.RecognizeCompleted":
-            selected_tone = event.data['dtmfResult']['tones'][0]
-            app.logger.info("Received DTMF tone %s", selected_tone)
-
-            if selected_tone == DtmfTone.ONE:
-                app.logger.info("Playing confirmed prompt")
-                call_connection_client.play_media_to_all([FileSource(CONFIRMED_PROMPT_URI)])
-            elif selected_tone == DtmfTone.TWO:
-                app.logger.info("Playing goodbye prompt")
-                call_connection_client.play_media_to_all([FileSource(GOODBYE_PROMPT_URI)])
-            else:
-                app.logger.info("Invalid selection, terminating call")
-                call_connection_client.play_media_to_all([FileSource(INVALID_PROMPT_URI)])
+            app.logger.info("Recognize completed: data=%s", event.data) 
+            if event.data['recognitionType'] == "choices": 
+                 labelDetected = event.data['choiceResult']['label']; 
```

Comment:
In Python it's recommended to use lowercase, with words separated by underscores 
https://peps.python.org/pep-0008/#function-and-variable-names